test/unit/modules/delegates.js

```diff
@@ -658,6 +658,96 @@ describe('delegates', () => {
 				);
 			});
 		});
+
+		describe('__private.delegatesListCache operations', () => {
+			let __private;
+			beforeEach(done => {
+				__private = library.rewiredModules.delegates.__get__('__private');
+				done();
```

Comment:
We are not waiting for any async operations here, you can get rid of `done();` and just `return;`

Reply:
That conflicts with eslint rules. See:
```
Unexpected synchronous test.
Unnecessary return statement.
```

Also I don't have a strong opinion on this but I'd prefer to keep it this way, since it's intuitively easier to understand and read. I mean when you look at the code it's easier to catch `done()` statements with eye than return statements. I didn't use done in tests to keep it consistent but the with architecture I'd like to use `done()`more often.